docs/starvingdead.rst

```diff
@@ -20,3 +23,20 @@ Usage
 
     starvingdead start
     starvingdead stop
```

Comment:
Since this is an "auto" script, there are a few more requirements, if you have the time. These types of scripts should now follow the enable API and persist their state (all copy-pasteable from existing scripts, so it shouldn't be that much more work).

docs are here for context: https://docs.dfhack.org/en/latest/docs/dev/Lua%20API.html#enabling-and-disabling-scripts

and you can totally just copy paste the code from `emigrate.lua`, which has very little business logic to get in the way.

The usage for this script should change to something like:

```
    enable starvingdead
    starvingdead [<options>]
```

where `starvingdead` run without options displays current status and configuration

Reply:
you'll know you've done the integration correctly if you run `enable` and `starvingdead` shows up with the correct status

Reply:
The docs do not mention it but you also need `--@module = true` in your document alongside `--@enable = true` for it to show up in the list... done though :)

Reply:
That's right. I'll have to add that to the docs. Thanks!